ext/opentelemetry-ext-pymysql/tests/test_pymysql_integration.py

```diff
@@ -17,22 +17,35 @@
 import pymysql
 
 import opentelemetry.ext.pymysql
-from opentelemetry.ext.pymysql import trace_integration
+from opentelemetry.ext.pymysql import PymysqlInstrumentor
 from opentelemetry.test.test_base import TestBase
 
 
 class TestPyMysqlIntegration(TestBase):
```

Comment:
It would be good to test tracer_provider could be provided when instantiating PymysqlInstrumentor and code take it correctly

Reply:
Good idea!, Done in https://github.com/open-telemetry/opentelemetry-python/pull/611/commits/c76ade0a0a2f9d2792d95856a7d04f0258b7e2e7 following the same implementation in mysql https://github.com/open-telemetry/opentelemetry-python/blob/5f188f777dccd82af3fa9b13a1142762efcfa708/ext/opentelemetry-ext-mysql/tests/test_mysql_integration.py#L41